WordPress/src/main/java/org/wordpress/android/ui/posts/PostListAction.kt

```diff
@@ -19,3 +23,45 @@ sealed class PostListAction {
     class ShowGutenbergWarningDialog(val site: SiteModel, val post: PostModel) : PostListAction()
     class DismissPendingNotification(val pushId: Int) : PostListAction()
 }
+
+fun handlePostListAction(activity: FragmentActivity, action: PostListAction) {
```

Comment:
It feels a little weird to have this function here, but I don't have a good suggestion for an alternative. I thought maybe we'd want to move it inside `PostListAction` but it doesn't bring significant benefit and since they are in the same file, it pretty much does the same thing. It also makes sure we can move this somewhere else if we wanted to mock it, so for now I think it's a good way to go. I just wanted to lay down my thoughts in case it's useful later.

Reply:
I agree this is not an ideal place. Let's move it somewhere else in the later phase of the project.